.github/CODEOWNERS

```
@@ -35,3 +35,6 @@
 
 # Release script
 /scripts/release.sh @hemilabs/dev @joshuasing
+
+# PoP payouts reporting service
+/popstats/ @gabmontes
```

As this repository is maintained by core-eng, I think requesting a review from core-eng would be a good idea for this directory as well:
```suggestion
/popstats/ @hemilabs/dev @gabmontes
```

---

Well, it looks like that defeats the purpose of the CODEOWNERS file. If `@hemilabs/dev` is owner of everything and you all are part of that team, everyone is owner of everything so no one has true ownership.

If `@hemilabs/dev` is added to `/popstats/` as suggested, anyone in that team can approve and merge changes without me having any say. Therefore, you should instead revisit how the rest of the `CODEOWNERS` file is defined.

---

You have a good point. We use code owners are more of a way to request reviews, and it isn't currently required through repo rulesets.

I thought this was additive, not one or the other, and it seemed to be this way for other entries (requesting a review from hemilabs/dev and someone else).
I think having both means that we can request a review from core-eng as well as you, meaning you will still need to review changes but everything gets a secondary review from core.

What are your thoughts?

---

Even in that case the usage pattern is weird. Let's say we have a team and a user as code owners, then a PR is created and both the team and the user are requested reviews. Now, if either of those (users within the team) or the individual user approves the PR, it goes through.

But as code owner reviews are not required by the repo config, then the CODEOWNERS config only helps with requesting reviews, none of which are mandatory.

Even when I prefer true ownership, I'm ok with the current setup if that works better for you guys.

cc/ @jcvernaleo @max-sanchez